sdk/webpubsub/Microsoft.Azure.Functions.Worker.Extensions.WebPubSub/src/Models/Output/WebPubSubAction.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.Azure.Functions.Worker
+{
+    /// <summary>
+    /// Abstract class of operation to invoke service.
+    /// </summary>
+    public abstract class WebPubSubAction
+    {
+        internal string ActionName
+        {
+            get
+            {
+                return GetType().Name.Replace("Action", "");
```

Comment:
Any reason why this can't be cached after the first call?

Reply:
It can be cached though it's called single time by the in-process extension. Let me update.

Reply:
After a few tests, the `ActionName` can't be cached that in the case single function call multiple actions, it will always resolve the init one which is not correct. So revert to get it each time.